cookies.css

```diff
@@ -1,181 +1,146 @@
-    background:white;
-    color: #000;
-    border-color: #00000071;
-    box-shadow: none;
-    opacity: 1;
-  }  
-  
-  div#cookies-modal.cookies-d-none {
-    display: none !important;
-  }
-
-  @media (min-width: 768px) {
-  
-    div#cookies-modal {
-      background: rgba(255, 255, 255, 0.65);
-      justify-content: space-between;
-      bottom: 20px;
-      border: 1px solid rgba(0, 0, 0, 0.18);
-      width: calc(100% - 10px);
-      border-radius: 8px;
-    }
-  
-    div#cookies-modal-buttons {
-      margin: 0;
-    }
-  
-  }
-
-  /*///////////// DARK MODE START /////////////*/
-
-  
-
-  div#cookies-modal.dark-theme {
-    background: rgba(0, 0, 0, 0.8);
-    border-top: 4px solid rgba(0, 0, 0, 1);
-  }
-
-  div#cookies-modal.dark-theme div#cookies-title, div#cookies-modal.dark-theme a, div#cookies-modal.dark-theme a:hover {
-    color: #fff;
-  }
-
-  div#cookies-modal.dark-theme button {
-    color: #fff;
-    background: rgba(0, 0, 0, 0.4);
-    border-color: #00000047;
-  }
-
-  div#cookies-modal.dark-theme button:hover, div#cookies-modal.dark-theme button#reject-cookies-button:hover {
-    background:rgba(0, 0, 0, 0.5);
-    color: #fff;
-  }
-
-  div#cookies-modal.dark-theme button#accept-cookies-button:before {
-    background-image: url('imgs/thumbs-up-regular-white.svg');
-  }
-
-  div#cookies-modal.dark-theme button#accept-cookies-button:hover:before {
-    background-image: url('imgs/thumbs-up-solid-white.svg');
-  }
-
-  @media (min-width: 768px) {
-
-    div#cookies-modal.dark-theme {
-      background: rgba(0, 0, 0, 0.6);
-      border: 1px solid rgba(0, 0, 0, 0.8);
-    }
-
-  }
-
-  /*///////////// DARK MODE END /////////////*/
-
-  /*///////////// CUSTOMIZE TYPOGRAPHY START /////////////*/
-
-  div#cookies-modal-container, div#cookies-modal-container div, div#cookies-modal-container a, div#cookies-modal-container button {
-    font-family: 'Poppins', Helvetica, Arial, sans-serif;
-  }
-
-  /*///////////// CUSTOMIZE TYPOGRAPHY END /////////////*/
-
-  /*///////////// CUSTOMIZE COLORS START /////////////*/
-  
-  div#cookies-modal a, div#cookies-modal a:hover {
-    border-color: #bfa290;
-  }
-
-  div#cookies-modal button#accept-cookies-button {
-    background: #bfa290;
-  }
-
-  /*///////////// CUSTOMIZE COLORS END /////////////*/
-  
-  /*///////////// COOKIES MODAL END /////////////*/
\ No newline at end of file
+dialog#cookies-modal {
+	align-content: center;
+	align-items: center;
+	background: rgba(255, 255, 255, 0.8);
+	border-bottom: none;
+	border-left: none;
+	border-right: none;
+	border-top: 4px solid rgba(0, 0, 0, 0.18);
+	bottom: 0px;
+	flex-direction: row;
+	flex-wrap: wrap;
+	font-family: "Poppins", Helvetica, Arial, sans-serif;
+	justify-content: center;
+	margin: 0 auto;
+	max-width: 900px;
+	padding: 10px 20px;
+	top: auto;
+
+	&[open] {
+		display: flex;
+	}
+
+	&::backdrop {
+		backdrop-filter: blur(8px);
+	}
+
+	&.dark-theme {
```

Comment:
Best thing about nesting is we keep things grouped logically without the need for comment blocks, for example all `dark-theme` styles are in this block.